packages/studio-plugin/src/http/ManagementApiService.ts

```diff
@@ -0,0 +1,82 @@
+import { HttpService } from "./HttpService";
+import { DEFAULT_API_VERSION, MANAGEMENT_API_ENDPOINTS } from "./constants";
+import {
+  EntitiesQueryParams,
+  EntitiesRequest,
+  EntitiesResponse,
+} from "./types/Entities";
+import { BaseQueryParams } from "./types/QueryParams";
+import { SavedFilterData } from "./types/SavedFilters";
+import ResponseParser from "./utils/ResponseParser";
+
+/**
+ * ManagementApiService handles making requests to the Management API for an
+ * account that matches the given API key.
+ */
+export default class ManagementApiService {
```

Comment:
We don't need to do this now, but would be nice to make this a separate package. Currently if we did that I think user's would need to install yet another dependency which we probably don't want. 
Eventually I think we'll figure out how to allow user's to get studio with just one npm package and then we could move this to a separate package kind of like search-core?

Reply:
I also like the idea of it being separate. That's one of the reasons why I wanted `http` to have its own `types` and `constants.ts`